plugin_tests/annotations_test.py

```diff
@@ -25,6 +25,11 @@
 
 from girder import config
 from girder.constants import AccessType
+from girder.models.collection import Collection
+from girder.models.folder import Folder
+from girder.models.item import Item
+from girder.plugins.large_image.models.annotation import Annotation
```

Comment:
Within the plugin test, imports from plugins have to be done in the test functions, not at the top of the file -- the plugin isn't present when the plugin test is imported.  Move the `from girder.plugins.large_image.models.annotation import Annotation` line to be inside each function where `Annotation()` is invoked (I'd add it just below each `def` as appropriate).

Reply:
Oh I didn't know that !! Thank you I will be able to fix the test and also other PR tests!!